examples/crewai_logging_example.ipynb

```diff
@@ -0,0 +1,118 @@
+{
```

Comment:
can the two new notebook be int he same agent folder (instead of one in autogen, one outside)